index.html

```diff
@@ -3,11 +3,44 @@
   <head>
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <link rel="stylesheet" href="/style.css"/>
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Vite App</title>
   </head>
   <body>
-    <div id="app"></div>
+    <div id="app">
+        <div id="modal-popup">
+            <table id="modal-popup-table">
```

Comment:
Try to investigate `flex` and `grid` in css. It is way more powerful tool than tables for that purpose